react/src/base/inputs/Checkbox.stories.js

```diff
@@ -1,23 +1,377 @@
 import React from 'react';
 import SprkSelectionInput from './SprkSelectionInput/SprkSelectionInput';
+import SprkErrorContainer from './SprkErrorContainer/SprkErrorContainer';
+import SprkCheckboxGroup from './SprkCheckboxGroup/SprkCheckboxGroup';
+import SprkCheckbox from './SprkCheckbox/SprkCheckbox';
+import SprkFieldset from './SprkFieldset/SprkFieldset';
+import SprkLegend from './SprkLegend/SprkLegend';
+import SprkHelperText from './SprkHelperText/SprkHelperText';
+import SprkStack from '../../objects/stack/SprkStack';
+import SprkStackItem from '../../objects/stack/components/SprkStackItem/SprkStackItem';
 import { markdownDocumentationLinkBuilder } from '../../../../storybook-utilities/markdownDocumentationLinkBuilder';
 
 export default {
   title: 'Components/Input/Checkbox',
-  decorators: [
-    story => <div className="sprk-o-Box">{story()}</div>
-  ],
-  component: SprkSelectionInput,
+  decorators: [(story) => <div className="sprk-o-Box">{story()}</div>],
+  component: SprkCheckboxGroup,
   parameters: {
+    subcomponents: {
+      SprkCheckbox,
```

Comment:
Is SprkCheckboxGroup supposed to be in the subcomponent list?

Reply:
Yes, because users need to know what variants are available.
![image](https://user-images.githubusercontent.com/4342363/85315972-f7416a00-b489-11ea-8a65-c17d67407889.png)